news/2017-01-19-bean-validation-2-0-progress-report.adoc

```diff
@@ -0,0 +1,235 @@
+This also gives us very good connections to the EGs of other specs such as JAX-RS or java.time (JSR 310) which will be beneficial for creating new (or improving existing) integrations with those.
+
+== First changes
+
+With the first EG members on board, we didn't lose time and began with the work on the new spec revision.
+One of the initial actions was to convert the spec document from DocBook into the fabulous http://asciidoc.org/[AsciiDoc] format.
+Using AsciiDoc comes with many advantages which make working on the spec a much more enjoyable experience:
+
+* It can be written using any text editor
+* Changes are easier to track, e.g. when reviewing pull requests on GitHub
+* We can include actual source files from the API instead of copying them
+
+While that's primarily a technicality interesting to those working on the spec, it also is beneficial for Bean Validation users,
+as you for instance can easily track all the changes done so far by examining a https://github.com/beanvalidation/beanvalidation-spec/compare/2a9d0ce21856386a8bf9a1d9e963ebffc049604a...spec-full[simple diff] on GitHub.
+
+== Support for new date and time API
+
+The primary theme in Bean Validation is the embrace of Java 8.
+Java 8 comes with a variety of improvements to the language (e.g. Lambda expressions and default methods)
+but also many useful additions to the class library.
+
+One prominent example of the latter is the new date and time API (https://www.jcp.org/en/jsr/detail?id=310[JSR 310]).
+Types such as `Instant`, `LocalDate` or `ZonedDateTime` are now supported by the `@Past` and `@Future` constraints (https://hibernate.atlassian.net/browse/BVAL-496[BVAL-496]):
+
+[source,java]
+----
+@Future
+private LocalDate deliveryDate;
+----
+
+`@Past` and `@Future` now also have a new attribute `orPresent()`:
+
+[source,java]
+----
+@Past(orPresent=true)
+private final Year inceptionYear = Year.of( 2017 );
+----
+
+That's useful for types such as `Year` or `LocalDate` which don't represent a specific instant but rather an interval of time
+and you want to consider the entire current year, day etc. as valid.
+
+Another improvement related to the validation of dates and times is the new `ClockProvider` extension point.
+It allows you to specify what is "now" when validating `@Past` and `@Future`.
+That comes in handy for instance if you want to work with the time and time zone of the currently logged in user in a multi-user, multi-timezone application.
+
+But it's also useful for (re-)running batch jobs with a different logical date than the current one or for testing with a fixed point in time considered as "now":
+
+[source,java]
+----
+Validator validator = Validation.byDefaultProvider()
+    .configure()
+    .clockProvider( () -> Clock.fixed(
+        Instant.parse("2017-01-19T11:00:00.00Z" ),  ZoneId.systemDefault() )
+    )
+    .buildValidatorFactory()
+    .getValidator();
+----
+
+== Validation of container elements
```

Comment:
This is not a good title for the audience this blog targets. It needs to mention concrete ones: `Collection` and `Optional`
You can mention later that it is generalized.